dist/api_data.py

```diff
@@ -688,6 +664,19 @@ def __ge__(self, other):
         cache_size and has to be greater than its counterpart \c eviction_updates_target. This
         setting only alters behavior if it is lower than \c eviction_trigger''',
         min=0, max='10TB'),
+    Config('extra_diagnostics', '[]', r'''
+        enable additional diagnostics in WiredTiger. These additional diagnostics include 
+        diagnostic assertions that can cause WiredTiger to abort when an invalid state 
+        is detected.
+        Options are given as a list, such as 
+        <code>"extra_diagnostics=[out_of_order,visibility]"</code>.
+        Choosing \c all enables all assertions. When WiredTiger is compiled with 
+        \c HAVE_DIAGNOSTIC=1 all assertions are enabled and cannot be reconfigured
+        ''',
+        type='list', choices=[
+            "all", "checkpoint_validate", "cursor_check", "eviction_check", 
+            "disk_validate", "hs_validation", "key_out_of_order", "log_validate", 
+            "log_validate", "prepared", "slow_operation", "txn_visibility"]),
```

Comment:
@clarissecheah it looks like we've added `log_validate` a second time instead of adding `generation_check`. Please check this and that everything is in alphabetical order

Reply:
Thanks for the catch! Fixed in [37a652d](https://github.com/wiredtiger/wiredtiger/pull/8696/commits/37a652d1d4d66c3aa8af24613f2c31c3e2b6b45a)